app/controllers/admin/collections_controller.rb

```diff
@@ -30,9 +26,23 @@ def index
       @department =  params[:department]
     end
 
+    scope = scope.order(Arel.sql("#{params[:sort_field]} #{params[:sort_order]}"))
```

Comment:
What if the `link_maker` handles this, so the knowledge about what the param keys are is in ONE place?

    scope = link_maker.add_order(scope)

Reply:
what do you think, nah?

Reply:
Oh this is also a security problem, passing `params` directly to `Arel.sql` like that opens up to SQL injection. Either need to use proper routinet o escape, or I think maybe you can use this to do it for you? `order(params[:sort_field] => params[:sort_order])`